it/test/uk/gov/hmrc/pillar2submissionapi/OverseasReturnNotificationISpec.scala

```diff
@@ -268,7 +270,7 @@ class OverseasReturnNotificationISpec extends IntegrationSpecBase with OptionVal
           "POST",
           submitUrl,
           INTERNAL_SERVER_ERROR,
-          Json.toJson(ORNErrorResponse("999", "internal_server_error"))
+          Json.toJson(ORNErrorResponse("500", "Internal Server Error"))
```

Comment:
Just a quick question (I know it's not your change). Why are we using `String` for the `ORDNErrorResponse` `code` and not `Int`?